stackslib/src/config/mod.rs

```diff
@@ -4244,6 +4258,8 @@ impl MinerConfigFile {
             max_execution_time_secs: self.max_execution_time_secs,
             replay_transactions: self.replay_transactions.unwrap_or_default(),
             stackerdb_timeout: self.stackerdb_timeout_secs.map(Duration::from_secs).unwrap_or(miner_default_config.stackerdb_timeout),
+            max_tenure_bytes: self.max_tenure_bytes.unwrap_or(miner_default_config.max_tenure_bytes),
+            log_skipped_transactions: false
```

Comment:
These skipped events might be nice for the explorer to show why a transaction is not being picked up. We might want to consider passing this option through the config file. If we don't do that, and only allow it to be set in tests, then why don't we remove this field from `MinerConfigFile`? My preference would be to support setting it in the config file.